randovania/games/dread/json_data/Ferenia.txt

```diff
@@ -1705,26 +1705,42 @@ Extra - asset_id: collision_camera_019
   * Layers: default
   * Morph Ball Tunnel to Energy Recharge Station Secret/Tunnel to Energy Recharge Station (Gate)
   > Tile Group (WEIGHT)
-      After Ferenia - grapplepulloff1x2_000 and Lay Bomb
+      All of the following:
+          After Ferenia - grapplepulloff1x2_000
+          Lay Bomb or Lay Power Bomb
 
 > Bomb Ledge; Heals? False
   * Layers: default
   > Dock to EMMI Zone Exit Middle
       Screw Attack
   > Event - Grapple Block (grapplepulloff1x2_000)
       All of the following:
-          Morph Ball
+          Grapple Beam and Morph Ball
           Any of the following:
               Cross Bomb
               All of the following:
                   # Bomb & Power Bombs, to desync the tunnel bomb blocks from the ledge bomb blocks
-                  Bomb and Power Bombs
+                  Bomb and Main Power Bomb and Power Bombs
+              All of the following:
+                  # Power bombs (video: https://twitter.com/i/status/1545111396760690692 )
+                  Main Power Bomb and Power Bombs ≥ 2 and Movement (Beginner)
+              All of the following:
+                  # Bombs (video: https://twitter.com/SantosJuice/status/1545703077138665472 )
+                  Bomb and Movement (Intermediate)
   > Tile Group (WEIGHT)
-      Morph Ball
+      Trivial
   > Tunnel to Energy Recharge Station Secret
       All of the following:
-          Morph Ball and After Ferenia - grapplepulloff1x2_000
-          Cross Bomb or Power Bombs
+          After Ferenia - grapplepulloff1x2_000
+          Any of the following:
+              Any of the following:
+                  # Without flash shift or bombs
+                  Lay Cross Comb or Lay Power Bomb
+                  Bomb and Morph Ball and Movement (Beginner)
```

Comment:
Is it possible to grab the ledge without spin or flash?

Reply:
Yeah, by simply unmorphing as you can see in my bombs only video  https://twitter.com/i/status/1545703077138665472. Timing is easier than destroying that block with a bomb.